.flowconfig

```diff
@@ -62,4 +62,4 @@ module.file_ext=.json
 module.file_ext=.jsx
 
 [version]
-^0.49.1
+^0.52.0
```

Comment:
RN 0.47 uses .0.49.1, the next one will use at least 0.53 as this is in their `master` but is not released yet.